scripts/tests/test_create_AR_DeployInst.py

```diff
@@ -281,6 +292,105 @@ def test_deploy_without_share(self):
                                       self.reactor_prototypes, 220)
         assert exp == obs
 
+    def test_deploy_without_share3(self):
+        '''
+        Test when the calculated number is negative'''
+        exp = 0
+        obs = di.deploy_without_share('Type1',
+                                      ['Type1','Type3','Type2'],
+                                      self.reactor_prototypes,
+                                      -100)
+        assert exp == obs
+
+    def test_redeploy_reactors1(self):
+        '''
+        Test when the calculated number is greater than the previous deployment
+        '''
+        exp = 2
+        deploy_schedule = {'DeployInst':{
+                            'prototypes':{
+                                'val':['Type1']},
+                            'n_build':{
+                                'val':[2]},
+                            'build_times':{
+                                'val':[0]},
+                            'lifetimes':{
+                                'val':[3]}
+                            }}
```

Comment:
```suggestion
        deploy_schedule = {'DeployInst':{
                            'prototypes':{
                                'val':['Type1']},
                            'n_build':{
                                'val':[2]},
                            'build_times':{
                                'val':[0]},
                            'lifetimes':{
                                'val':[3]}
                            }}
```

This pattern gets repeated several times, which means you should either
* define the dictionary globally
* or create a [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html).